loony_dev/github/issue.py

```
@@ -93,32 +99,55 @@ def get(cls, number: int, *, repo: Repo) -> Issue:
+        )
+
         return cls(
             number=data["number"],
-            title=Content(data.get("title", "")),
-            body=Content(data.get("body", "")),
+            title=Content(title_str, safe=sanitize_content),
+            body=Content(body_str, safe=sanitize_content),
             author=data.get("author", {}).get("login", ""),
             updated_at=parse_datetime(data.get("updatedAt")),
+            created_at=parse_datetime(data.get("createdAt")),
             labels=[l["name"] for l in data.get("labels", [])],
```

Ruff (0.15.10)

[error] Ambiguous variable name: `l`

(E741)